MPF.md

```diff
@@ -643,6 +643,59 @@ Here is an example setting the acl from augments:
 
 **See Also:** [Configure networks allowed to make collect calls (client initiated reporting)](#configure-networks-allowed-to-make-collect_calls-client-initiated-reporting)
 
+### Configure hosts that may connect to cf-serverd
+
+`allowconnects` is a list of IP addresses or subnets in `body server control` restricts hosts that are allowed to connect to `cf-serverd`. This is the first layer of access control in `cf-serverd`, a client coming from a host not covered by this list will not be able to connect to `cf-serverd` at all.
+
+In the MPF this defaults to include localhost and the value defined for `default:def.acl`.
+
+`allowconnects` can be customized by configuring `default:def.control_server_allowconnects` via Augments.
```

Comment:
so if you define this `allowconnects` does it clobber the previous "localhost and default:def.acl" or add to it?

Reply:
ah, now I see the code below, it "overwrites" you might mention that here in the docs.

Reply:
Yeah, I was a bit on the fence about extend vs overwrite when I started. I asked for opinions there but I didn't wait long before I made a choice and proceeded:

With full definition you can achieve extension by re-defining the defaults. With extension you can't really influence the defaults that include localhost and def.acl, so I guess full definition seems the best choice.

Adding a note as suggested.